src/css/components/app-health-detail.less

```diff
@@ -20,7 +20,7 @@
 .health-breakdown-item {
   margin-left: @base-spacing-unit;
   &.health-breakdown-item-unknown {
-    color: @task-health-unknown-color;
+    // color: @task-health-unknown-color;
```

Comment:
We don't want comments in our code – let's kick the whole line (and if appropriate, the whole block) out. Thanks!